### ISSUE TITLE
Sign In page

### ISSUE THREAD

**Issue by cromdenne** [mswysocki/Madison-Student-Apartment-Finder#10]
There's a small problem that I can't seem to find the source of.  On the sign in page, if a sign in is unsuccessful then a message pops up at the bottom of the page.  However, the message appears outside of the content area, so you can't really see it.  It should be really easy to fix, I just can't find where in the html that error is being generated.

**Comment by cromdenne** [comment]
A similar issue arises when you create a new user.  There is a welcome message after the user has been created, but the welcome message is outside of the content area so you can can't really tell it's there.

**Comment by aeggum** [comment]
Yeah, those are a ruby/rails thing called flash messages.  They are actually generated in the controllers, if I recall correctly.  I think that it automatically generates the message in a class called "flash error", when looking at the source.  Obviously the problem is that they are generated outside of the < section > tags that they need to be in.   

I'm not sure if it will help, but I think the code that deals with it in the html itself is inside the layouts/application.html.erb file.  If it can't be in there maybe it can be moved to each file that we have to deal with it in, I'm not sure.  

Also, I just fixed some problems with the debug statements at the bottom of the file.

Hope this helps!

**Comment by jds0102** [comment]
There is also a flash message now that appears after a user requests to have their password reset.
